askomics/libaskomics/source_file/SourceFile.py

```diff
@@ -533,7 +533,7 @@ def load_data_from_file(self, fp, urlbase):
         url = urlbase+"/ttl/"+os.path.basename(fp.name)
         data = {}
         try:
-            if self.is_defined("askomics.file_upload_url") and "8890" not in self.get_param('askomics.file_upload_url'):
+            if self.is_defined("askomics.file_upload_url"):
```

Comment:
Before I used "8890" to search if the endpoint is Virtuoso because this function doesn't work with this triplestore.

But I removed this line because the endpoint of Virtuoso could be changed. 
Instead of I put some comments in [configs files for Fuzeki](https://github.com/askomics/askomics/pull/13/commits/a264a128d4e4f3fedc047050eea5024871cdf8d7#diff-08a25def326cf98721b26d5ce907cc85R8) and in [QueryLauncher.py](https://github.com/askomics/askomics/pull/13/commits/dff2624e1f434e7da98a788dbaa6dfb48ba5fef7#diff-f1595749d135659d6b89fd19880fd3abR150) preventing the developer who want to add an 'askomics.file_upload_url' in the Virtuoso configs files.